.travis.yml

```diff
@@ -26,8 +27,7 @@ before_install:
 
 install:
 - export PATH=`pwd`/texlive/bin/x86_64-linux:$PATH
-- npm install
-- grunt setup
+- .travis/prepare_npm
```

Comment:
I guess the script can be omitted if you just write:

`- find . -name package.json ! -path '*/node_modules/*' -execdir npm install \;`

A big nitpicker, though ;-)

Reply:
You could be right 
But i prefer the script:
1. it can be locally tested
2. i use it in the local build script run_local_build.sh to test
3. it‘s my first bash script 😉

If you could agree could you do the merge please?

Reply:
OK, got it. wonderful find. read the man page. Could you live with the script? It echos the working dir.

Reply:
I didn't block the PR because of that - that was just a hint that the script isn't needed in this case. The single `find` is more robust against spaces in the path and it spawns much less processes than the script with it's loop. Both facts are not relevant in this case, so I'm fine with the script.